src/components/Global/Header/Header.module.scss

```diff
@@ -0,0 +1,76 @@
+$white: white;
+$black: black;
+$primary-color: #F16B67;
+
+.header {
+    width: 100vw;
+    height: 7vh;
+    background-color: $white;
+    padding: 0 2vw;
```

Comment:
Header should be a fixed height (rem based). If you shrink the window height the header will shrink which could make it unusable.